#ifndef __PROTOBUF_H__
#define __PROTOBUF_H__

#include "protocol/10000_C2SRobotLoginReq.pb.h"
#include "protocol/10001_S2CRobotLoginRsp.pb.h"
#include "protocol/20000_C2SQQLoginReq.pb.h"
#include "protocol/20001_S2CQQLoginCaptchaNotify.pb.h"
#include "protocol/20002_C2SQQLoginCaptchaConfirm.pb.h"

#endif